substrate/frame/staking/src/pallet/impls.rs

```diff
@@ -1028,6 +1058,27 @@ impl<T: Config> Pallet<T> {
 			DispatchClass::Mandatory,
 		);
 	}
+
+	/// Temporary getter for `Payees`.
+	///
+	/// Migrates `Payee` to `Payees` if it has not been migrated already.
+	pub fn get_payout_destination_migrate(
+		stash: &T::AccountId,
```

Comment:
Would it not be easier to make `Payees` be `OptionQuery`? I think it was already confusing that `Payee` was not `OptionQuery`.

Reply:
Would it be reasonable for  an `OptionQuery` of `None` to act as the `Forgo` variant?